packages/modules/src/modules/banners/common/choiceCard/ChoiceCardsSupportCta.tsx

```diff
@@ -13,45 +11,27 @@ const buttonOverrides = css`
 `;
 
 export const ChoiceCardsSupportCta = ({
-    tracking,
-    numArticles,
-    countryCode,
-    amountsTestName,
-    amountsVariantName,
-    selection,
     getCtaText,
+    getCtaUrl,
     cssOverrides,
     onCtaClick,
 }: {
-    tracking: Tracking;
-    numArticles: number;
-    countryCode?: string;
-    amountsTestName?: string;
-    amountsVariantName?: string;
-    selection: ChoiceCardSelection;
-    getCtaText: (contentType: 'mainContent' | 'mobileContent') => string;
+    getCtaText: (contentType: ContentType) => string;
+    getCtaUrl: (contentType: ContentType) => string;
     cssOverrides?: SerializedStyles;
     onCtaClick: () => void;
 }): JSX.Element | null => {
-    const url = `https://support.theguardian.com/contribute?selected-contribution-type=${selection.frequency}&selected-amount=${selection.amount}`;
-
     const mobileText = getCtaText('mobileContent');
     const desktopText = getCtaText('mainContent');
 
-    const supportUrl = addRegionIdAndTrackingParamsToSupportUrl(
```

Comment:
This is already done from the `BannerWrapper` when we use the configured url - https://github.com/guardian/support-dotcom-components/blob/main/packages/modules/src/modules/banners/common/BannerWrapper.tsx#L166